source/_includes/pup43_platforms_fedora.markdown

```diff
@@ -0,0 +1,7 @@
+We publish official packages and run automated testing on the following versions for Puppet 4.3:
```

Comment:
OK, but I'm afraid I don't follow; this language isn't new, and if we change it here, we need to reassess how we define "Puppet" and "packages" throughout the docs.

Are you saying we should specify `puppet-agent` and `puppetserver` packages here, or Puppet Collections, or something else? Or that we shouldn't use the word "official"?